### PR TITLE
PLAT-5595: Modularize the scaffolds - Modules

### PR DESCRIPTION
**Pull request by santoshpy**
## Ticket

PLAT-5595
_Related tickets:_ [PLAT-5595](https://crowdbotics.atlassian.net/browse/PLAT-5595)
_Related PRs:_

## Type of PR

- [ ] Bugfix
- [X] New feature
- [ ] Minor changes

## Changes introduced

Manage the folder structure for backend and mobile scaffold. 
Update README.md file for consistency 

## Test and review

AS this PR is related to PLATFORM-DASHBOARD to test it, Need to merge all related PR.

### CODE REVIEW

**Review by danielsousaio — COMMENTED** [review]
Please consider the related review on [crowdbotics/django-scaffold](https://github.com/crowdbotics/django-scaffold/pull/248#pullrequestreview-747356162).

File: dist/cookie/{{cookiecutter.project_slug}}/README.md [hunk 1]
```diff
@@ -40,32 +55,145 @@ As a result, Crowdbotics can generate new applications automatically in a standa
...
 ### Metro
 
 After cloning the repo, install the dependencies locally with [Yarn](https://yarnpkg.com/):
 
+Goto `/frontend` and run `yarn install`
+
+```sh
+cd {{cookiecutter.project_slug}}/frontend
+```
+
```
Comment (danielsousaio) [inline dist/cookie/{{cookiecutter.project_slug}}/README.md:196]: Should this read `/mobile`?

File: dist/raw/ProjectName/README.md [hunk 2]
```diff
@@ -1,37 +1,52 @@
-# ProjectName
+# {{cookiecutter.project_slug}}
```
Comment (mabdullahabid) [inline dist/raw/ProjectName/README.md:1]: @danielsousaio would changing the ProjectName to cookiecutter slug work here? I don't think so. Can you confirm?

File: scaffold/template/custom/README.md [hunk 3]
```diff
@@ -1,37 +1,52 @@
-# ProjectName
+# {{cookiecutter.project_slug}}
```
Comment (mabdullahabid) [inline scaffold/template/custom/README.md:1]: @danielsousaio same here

**Review by mabdullahabid — CHANGES_REQUESTED** [review]
The logic for using the mobile folder and handle all subsequent actions after in `scaffold/install.js` looks okay to me. However, I am not able to review the correctness of the syntax of this JavaScript file.

@danielsousaio can you confirm if this looks good?

@santoshpy There are merge conflicts (which is expected) so maybe you should merge master into this branch and then handle conflicts accordingly.

**Comment by driverdan** [comment]
@santoshpy Please fix the merge conflicts

**Comment by danielsousaio** [comment]
Closing stale PR

**Pull request closed without merging**